### ISSUE TITLE
Bot Run assignment issues with VoiceOver

### ISSUE THREAD

**Issue by howard-e** [w3c/aria-at-app#1270]
Bot run oddities observed during [today's CG meeting](https://www.w3.org/2024/11/06-aria-at-minutes.html#9f76):

1. Admin @IsaDC assigned `VoiceOver Bot` to "Action Menu Button Example Using element.focus() - V24.10.31 for VoiceOver + Safari". The bot reported 1 error and 10 tests cancelled, for the 11 tests in the test plan. [[View logs on that failed run]](https://github.com/bocoup/aria-at-gh-actions-helper/actions/runs/11618336761).
2. Afterwards, the results of that bot run was assigned to Isa which expectedly had no results populated, to be evaluated.
3. Isa's run was then removed and a new run created. It was then immediately assigned to IsaDC to be evaluated. This run was successful but it was able to be assigned before the run was completed. That doesn't impact the results being populated, but this is a particularly confusing case when the expectation is that a bot's results can only be assigned and evaluated AFTER the bot run has been completed [[View logs on that successful run]](https://github.com/bocoup/aria-at-gh-actions-helper/actions/runs/11709768478/job/32614530609)

TODO:
1. Investigate why the VoiceOver bot failed (logs above in no.1).
2. Investigate @mcking65's report on why the 'Manage Bot Run' button didn't appear to get to the option to re-run the bot's results collection.
3. Prevent the bot's run from being assigned until it is fully completed.

**Comment by ccanash** [comment]
Closing this issue as it addressed items 2 and 3.  Opened https://github.com/w3c/aria-at-app/issues/1305 to track item 1 (cc @IsaDC )
